envConfig/live.env

```diff
@@ -1,9 +1,10 @@
+## NB only variables prefixed with SIMORGH_ are available on both the server and client
```

Comment:
![source](https://user-images.githubusercontent.com/8601460/72205086-f1754080-3476-11ea-932d-5e2f7012910b.gif)

That explains the bug at least - hadn't realised the `isLive` function we were using was added only for this feature.  So I guess it **was** returning the wrong value, but didn't have any other effect because it wasn't used anywhere else?